sale_multi_operating_unit/models/res_company.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    @api.model
+    def _get_default_lead_description_template(self):
+        return
+        """
```

Comment:
Hey @max3903 This method return none so I guess **lead_description_template** field not set any default value here. Can you specify some default value here please or let me know so I can set default value here,

Reply:
Fixed